runtime/frequency/src/lib.rs

```diff
@@ -202,6 +206,10 @@ impl frame_system::Config for Runtime {
 	/// The identifier used to distinguish between accounts.
 	type AccountId = AccountId;
 	/// Base call filter to use in dispatchable.
+	// enable for cfg feature "frequency" only
+	#[cfg(not(feature = "frequency"))]
```

Comment:
This worries me `not(feature)` can be an issue for `--all-features` in testing, benchmarking, and checks.

Any ideas about other ways to handle it? I wonder if this code needs to use a macro like `prod_or_testnet_or_local_or_env`

Reply:
So the order in which flags are defined are ```not frequency ```and then ``` frequency``` . This entails when a binary is built with all-features flag it will always choose first over the other

And mainnet binary will not see ```not part``` that way we default rococo(local, sealing, testnet) if binary is built with all-features 

When we release mainnet we build using frequency flag and it will only keep mainnet related definitions

Reply:
I will put a macro and check